src/compat/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.12.0)
+
+execute_process(
+  COMMAND "/usr/bin/git" clone -b main https://github.com/felixguendling/rp-compat-bin.git bin
+  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+)
+
+execute_process(
+  COMMAND "/usr/bin/git" clone -b main https://github.com/felixguendling/rp-compat-corpus.git corpus
```

Comment:
can't this simply be in the main 3rd_party_deps.cmake with a custom `install` step?